### ISSUE TITLE
ValueError: invalid vertex: 3; condense-edges.py error.

### ISSUE THREAD

**Issue by pavlohrab** [fpusan/SuperPang#13]
Hi,

I was trying to run Superpang on a 50 MAGs (supposedly 1 species) . It returned the following error:

```
Traceback (most recent call last):
  File "/lustre/BIF/nobackup/hrab001/micromamba/envs/superpang/lib/python3.8/site-packages/superpang/scripts/condense-edges.py", line 278, in <module>
    main(parse_args())
  File "/lustre/BIF/nobackup/hrab001/micromamba/envs/superpang/lib/python3.8/site-packages/superpang/scripts/condense-edges.py", line 222, in main
    predecessors, successors = G2dicts(GS, name2vertex, vertex2name)
  File "/lustre/BIF/nobackup/hrab001/micromamba/envs/superpang/lib/python3.8/site-packages/superpang/scripts/condense-edges.py", line 238, in G2dicts
    for pv in GS.get_in_neighbors(v):
  File "/lustre/BIF/nobackup/hrab001/micromamba/envs/superpang/lib/python3.8/site-packages/graph_tool/__init__.py", line 2441, in get_in_neighbors
    vertices = libcore.get_in_neighbors_list(self.__graph, int(v),
ValueError: invalid vertex: 3

There was an error running condense-edges.py. Please open an issue
```

Can you advice on resolving it?

Best,
Pavlo

**Comment by fpusan** [comment]
Is this happening with the latest version of SuperPang?
Can you share those 50 genomes with me so I can try to replicate?

**Comment by fpusan** [comment]
Managed to reproduce this.
`mamba install -c conda-forge graph-tool==2.45` will fix the bug for you. I will try to find a more proper fix and release a patch ASAP

**Comment by fpusan** [comment]
I just released version `1.3.0.post1`, which adds compatibility with newer graph-tool versions.
